src/pylon_camera/pylon_camera_parameter.cpp

```diff
@@ -204,6 +207,16 @@ void PylonCameraParameter::readFromRosParameterServer(const ros::NodeHandle& nh)
             }
         }
     }
+
+
+    
+    if ( nh.hasParam("whitebalance_auto") )
```

Comment:
Dynamic reconfigure actually does this. It checks for a ROS parameter with the name of the parameter at startup. So if you set it in the launchfile, dynamic reconfigure will pass that value to your callback.